ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsTableQueries.scala

```diff
@@ -22,4 +22,18 @@ object EventsTableQueries {
     lastEventNodeIndexFromPreviousPage
       .map(x => (lastOffsetFromPreviousPage, x))
       .getOrElse(NonExistingOffsetNodeIndexPair)
+
+  def format(ps: Set[Party]): String =
+    ps.view.map(p => s"'$p'").mkString(",")
```

Comment:
This formatting should already be applied by Anorm and should cache queries when possible, any reason to not use that?

Reply:
I just saw that you use this to specify the array, marking as resolved.